### ISSUE TITLE
TravisCI failes due to terminated VM

### ISSUE THREAD

**Issue by lgoltz** [deegree/deegree3#729]
The last TravisCI Jobs failed or errored. The following exception can be found in the log file:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.16:test (default-test) on project deegree-core-base: Execution default-test of goal org.apache.maven.plugins:maven-surefire-plugin:2.16:test failed: The forked VM terminated without saying properly goodbye. VM crash or System.exit called ?
[ERROR] Command was/bin/sh -c cd /home/travis/build/deegree/deegree3/deegree-core/deegree-core-base && /usr/lib/jvm/java-7-openjdk-amd64/jre/bin/java -jar /home/travis/build/deegree/deegree3/deegree-core/deegree-core-base/target/surefire/surefirebooter7010094970586568024.jar /home/travis/build/deegree/deegree3/deegree-core/deegree-core-base/target/surefire/surefire2083261284484003695tmp /home/travis/build/deegree/deegree3/deegree-core/deegree-core-base/target/surefire/surefire_5602126721869629010tmp
```

**Comment by lgoltz** [comment]
The problem is described in https://github.com/travis-ci/travis-ci/issues/5631 Increasing the Java heap size in the surefire plugin should solve this problem: 
```xml
    <plugins>
        <plugin>
            <groupId>org.apache.maven.plugins</groupId>
            <artifactId>maven-surefire-plugin</artifactId>
            <configuration>
                <!-- Travis build workaround -->
                <argLine>-Xms1024m -Xmx2048m</argLine>
            </configuration>
        </plugin>
    </plugins>
```

**Comment by lgoltz** [comment]
All checks for the PR #730 passed, so the workaround seems to fix the problem.

**Comment by tfr42** [comment]
Fix applied for 3.4 with PR #730 and for 3.3 with PR #732.
